src/Packetery/Module/Order/GridExtender.php

```diff
@@ -455,4 +468,28 @@ public function makeOrderListSpecificColumnsSortable( array $columns ): array {
 
 		return wp_parse_args( [ 'packetery_packet_stored_until' => $metaKey ], $columns );
 	}
+
+	/**
+	 * Gets the size in the configured dimension unit.
+	 *
+	 * @param Core\Entity\Order $order Order entity.
+	 *
+	 * @return Size
+	 */
+	public function getSizeInSetDimensionUnit( Core\Entity\Order $order ): Size {
```

Comment:
Název by měl začínat na `create`.
Je to už druhá metoda na vytváření objektu Size. Co udělat factorku?
Tím se alespoň ospravedlní, že je public ;)

Reply:
Vytvoreno.